spring-cloud-kubernetes-integration-tests/run.sh

```diff
@@ -137,12 +177,15 @@ main() {
 
 run_tests() {
 	arr=("$@")
```

Comment:
I need some closer eyes here, on this part:

```
cd ../spring-cloud-kubernetes-test-support
${MVN} clean install
cd ../spring-cloud-kubernetes-integration-tests
```
I did some changes in `spring-cloud-kubernetes-test-support`; for example I made `static` some methods that where non-static. When I try to run an integration test _without_ the portion of the code above, I get : "can not references static method from non-static...". It's like i am running the previous version of the code, _before_ I made changes to it. 

Initially I thought this has to do with circleci caching, but I after some testing - this was not the case. I am wondering if `spring-cloud-kubernetes-test-support` is somehow pushed to your Spring's nexus repository... currently I am a bit in the blur. I'll leave this comments as is, until I figure out  what is going on

Reply:
Yes it is https://repo.spring.io/ui/packages/gav:%2F%2Forg.springframework.cloud:spring-cloud-kubernetes-test-support?name=spring-cloud-kubernetes-test-support&type=packages

So this makes sense to me, I would add a comment about it though